app/uk/gov/hmrc/datecalculator/services/WorkingDaysService.scala

```diff
@@ -166,8 +177,8 @@ class WorkingDaysService @Inject() (
     }
   }
 
-  private def getAndSetBankHolidays()(implicit hc: HeaderCarrier): Future[BankHolidays] =
-    bankHolidaysService.getBankHolidays().map{ bankHolidays =>
+  private def getAndSetBankHolidays()(using hc: HeaderCarrier): Future[BankHolidays] =
```

Comment:
Same here for hc

Reply:
fixed [here](https://github.com/hmrc/date-calculator/pull/17/commits/d43ef9ce930b0f314bf79c842493444eeacdd550)